templates/taxi/driver_list.html

```diff
@@ -1,30 +1,34 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+  <h1>Driver List</h1>
 
-    {% if driver_list %}
+  {% if driver_list %}
```

Comment:
Code Style: According to the checklist, you should use 2 whitespaces indentations in your `.html` files. Please adjust your indentation accordingly.